crates/aptos/src/ffi.rs

```diff
@@ -1,84 +0,0 @@
-// Copyright © Aptos Foundation
```

Comment:
I can't even remember, what was this for?

Reply:
It lets you use dynamic library linking to TS at the time.

Pointless, as we have a replacement in just shelling out or properly supporting wasm.